### DISCUSSION TITLE
Option to invert the behavior Menu Entry Swapper Shift Click to Walk here

### DISCUSSION THREAD

**Discussion by Liimurr** [runelite/runelite#16466] [Ideas]
![image](https://user-images.githubusercontent.com/14037388/230492559-095a43f4-06db-4f51-a337-b141287cf901.png)

A checkbox to invert the behavior of this feature. 

If checked the user would normally walk over items/npcs and would interact/attack/pickup when holding shift.

**Comment by Nightfirecat** [comment]
Why would anyone want this?

**Reply by Liimurr** [reply]
Hey! I can think of 3 use cases. In each case the user would not want the regular behavior the majority or 100% of the time.

Use case 1:
As a skiller training on 2 tick teak logs I would want this feature so that I can do the following without pressing/holding shift while training. 
- walk on the same tile as the bunnies/birds used for tick manipulation when resetting their attack cycles after a missclick
- avoid talking to random event npcs
- avoid picking up teak logs dropped at your feet


Use case 2: 
As an Herb runner I would want this feature so that I can do the following without pressing/holding shift while doing an herb run:
- avoid picking up buckets/weeds dropped by other players
- avoid interacting with farmers that may wonder over to the herb patch

Use case 3:
As a birdhouse runner I would want this feature to avoid accidentally picking up items without pressing/holding shift while doing the run.

In general, a user may want this feature if they engage in activities like the above where they would be avoiding picking up items or walking on the same tile as NPCs over 50% of the time for an extended period of time.
